src/buildSlice.ts

```diff
@@ -236,11 +241,39 @@ export function buildSlice({
     },
   });
 
+  const configSlice = createSlice({
+    name: `${reducerPath}/config`,
+    initialState: {
+      online: true,
```

Comment:
can we assume this?

Reply:
I'm using the `isOnline` util now. Behaviors are tested in `utils.test`